DiscordBot.Commands/SoundCommands/SoundUploadCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Discord.WebSocket;
+using DiscordBot.Commands.Core.CommandAttributes;
+using DiscordBot.Commands.Core;
+using Microsoft.Extensions.Logging;
+using Discord;
+
+namespace DiscordBot.Commands.SoundCommands
+{
+    [Name("soundupload")]
+    [Description("For uploading new sounds for bot to play")]
+    [Option("file", "File in mp3 fromat", CommandOptionType.Attachment)]
+    public class SoundUploadCommand : AbstractSoundCommand
+    {
+        private readonly ILogger<SoundUploadCommand> _logger;
+
+        public SoundUploadCommand(ILogger<SoundUploadCommand> logger)
+        {
+            _logger = logger;
+        }
+
+        public override async Task ExecuteAsync(SocketSlashCommand command)
+        {
+            command.DeferAsync().Wait();
```

Comment:
Why `Wait()`?

Reply:
In youtube command it is made the same way. And probably it should `Wait()` as discord should acknowledge defer, so the code as trying to modify response could throw some errors.

Reply:
@Niewidzialny84 I see that in youtube command the function is not actually marked as `async` and probably that's why it there this way. That's a separate issue, here (and in the rest of the commands) the `DeferAsync` can should be properly awaitable.